packages/dashmate/test/status/mocks/mnSyncMock.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
We usually have "global" mocks dir, like `packages/dashmate/test/mocks/status/mnSyncMock.json` so other tests can use them not just status.